### PR TITLE
feat(server): Implement Api Get Video Trending

### LINKED ISSUE #31: Server: Api Get Trending Youtube Music

### PR DESCRIPTION
**Pull request by trungpham71198**
Closes #31

### CODE REVIEW

File: apps/server/src/datasources/utils/controllers/youtube/getVideoTrending.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+export default async function getVideoTrending(): Promise<Array<Song>> {
+  try {
+    const songs: Song[] = [];
+
+    const data = await getData({
+      urlString: 'https://www.youtube.com/youtubei/v1/browse',
+      method: 'POST',
+      reqBody: {
+        browseId: 'FEtrending',
+        params: '4gINGgt5dG1hX2NoYXJ0cw%3D%3D',
```
Comment (khoaquocngo) [inline apps/server/src/datasources/utils/controllers/youtube/getVideoTrending.ts:15]: add to env

File: apps/server/src/datasources/utils/controllers/youtube/getVideoTrending.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      },
+      reqHeaders: {
```
Comment (khoaquocngo) [inline apps/server/src/datasources/utils/controllers/youtube/getVideoTrending.ts:17]: add to config

File: apps/server/src/datasources/utils/controllers/youtube/helpers/getData.ts [hunk 3]
```diff
@@ -25,20 +25,35 @@ export default async function getData(urlString: string) {
...
+
   let headers;
   if (isAjax) {
     const data = {
       context: { client: { clientName: 'WEB', clientVersion: '2.20210401.08.00' } },
       continuation: url.searchParams.get('token'),
+      ...reqBody,
     };
-    body = (await axios({ method: 'post', url: urlString, data: data })).data;
+    headers = {
```
Comment (khoaquocngo) [inline apps/server/src/datasources/utils/controllers/youtube/helpers/getData.ts:40]: refactor headers

File: apps/server/src/datasources/utils/controllers/youtube/getVideoTrending.ts (hunk 2 above)
Comment (trungpham71198) [reply apps/server/src/datasources/utils/controllers/youtube/getVideoTrending.ts]: done!

**Review by khoaquocngo — APPROVED** [review]

**Pull request merged**